tyk-headless/templates/_helpers.tpl

```diff
@@ -30,3 +30,23 @@ Create chart name and version as used by the chart label.
 {{- define "tyk-headless.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "tyk-headless.redis_url" -}}
+{{- if .Values.redis.addrs -}}
+{{ join "," .Values.redis.addrs }}
+{{- else if and .Values.redis.host .Values.redis.port -}}
+{{ .Values.redis.host }}:{{ .Values.redis.port }}
+{{- else -}}
+redis.{{ .Release.Namespace }}.svc.cluster.local:6379
+{{- end -}}
+{{- end -}}
+
+{{- define "tyk-pro.mongo_url" -}}
+{{- if .Values.mongo.mongoURL -}}
+{{ .Values.mongo.mongoURL }}
+{{- else if and .Values.mongo.host .Values.mongo.port -}}
+mongodb://{{ .Values.mongo.host }}:{{ .Values.mongo.port }}
```

Comment:
Doesn't it should include the database `tyk_analytics` as part of the connection string?

Reply:
Correct. Fixed.